forc-plugins/forc-client/src/util/tx.rs

```diff
@@ -311,6 +309,8 @@ impl<Tx: Buildable + field::Witnesses + Send> TransactionBuilderExt<Tx> for Tran
                 e
             })?;
 
+        // Set max_fee from consensus_parameters.
```

Comment:
Is that what the section below is doing? I don't see a reference to `max_fee` in lines 314-326

Reply:
Opps nice catch thanks